yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -809,6 +809,8 @@ func fetchUnsupportedPGFeaturesFromSchemaReport(schemaAnalysisReport utils.Schem
 	addUnsupportedFeaturesFromSchemaAnalysisReport("Conversion objects", CONVERSION_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
 	addUnsupportedFeaturesFromSchemaAnalysisReport("Gin Indexes on Multi-columns", GIN_INDEX_MULTI_COLUMN_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
 	addUnsupportedFeaturesFromSchemaAnalysisReport("Unsupported DDL operations", ADDING_PK_TO_PARTITIONED_TABLE_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
+	addUnsupportedFeaturesFromSchemaAnalysisReport("Foreign tables", FOREIGN_TABLE_ISSUE_REASON, schemaAnalysisReport, &unsupportedFeatures)
```

Comment:
I don't think we should be classifying this as an unsupported feature. It is supported in YB. It's just that you need to configure the prerequisites yourself. 

Shall we just add a note to the assess-migration report saying something along the lines of - "you have some foreign tables. after running export-schema, you will have to do so and so manually. For more information, you can run analyze-schema."?

Reply:
Hmm, that makes sense. Will make the change

Reply:
Made the change and added sample in description